service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseManagementOrderProgressionServiceTest.java

```diff
@@ -50,11 +54,19 @@
     @Autowired
     private ObjectMapper mapper;
 
+    @Mock
+    private ApplicationEventPublisher applicationEventPublisher;
+
+    @Mock
+    private RequestData requestData;
+
     private CaseManagementOrderProgressionService service;
 
     @BeforeEach
     void setUp() {
-        this.service = new CaseManagementOrderProgressionService(mapper);
+        given(requestData.authorisation()).willReturn("Bearer");
```

Comment:
Is this and the line below needed?

Reply:
Actually I think it is, my tests were failing without it because requestData wasn't mocking properly (couldn't call authorisation() because test thought requestData was null). I'll double check though

Reply:
Nevermind - not needed 👍